app/src/main/res/layout/activity_main.xml

```diff
@@ -4,7 +4,7 @@
     xmlns:tools="http://schemas.android.com/tools"
```

Comment:
Changes made to any res files should only be because I added an activities folder and put the Java MainActivity and TimerActivity in there. AndroidStudio updated the file path when it references MainActivity.